core/templates/dev/head/filters.js

```diff
@@ -465,6 +467,20 @@ oppia.filter('stripFormatting', [function() {
   };
 }]);
 
+oppia.filter('getAbbreviatedText', [function () {
+  return function (text, characterCount) {
+    if (text.length > characterCount) {
+      var subject = text.substr(0, characterCount);
+
+      if (subject.indexOf(' ') > -1) {
```

Comment:
Consider !== rather than >. They do the same thing but conceptually -1 is a sentinel value here rather than "one position less than 0".

Reply:
Done